ui-tests/src/test/java/org/openmrs/reference/DuplicatePatientRegisterTest.java

```diff
@@ -69,19 +66,15 @@ public void duplicateRegisterTest() throws InterruptedException, ParseException
         sdf.applyPattern(NEW_FORMAT);
         newBirthDate = sdf.format(date);
 
-
         String info = registrationPage.getSimilarPatientInfo();
         assertThat(info, Matchers.is(patient.gender + ", " + newBirthDate + ", " + patient.address1 + " " + patient.address2 + " " + patient.city + patient.state + patient.country + patient.postalCode ));
     }
 
-
```

Comment:
Get rid of these spaces

Reply:
> Get rid of these spaces

Have you had a look at the second commit that emphasizes quality assurance